app-vcs/gitea/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=gitea
+PKGSEC=vcs
+PKGDES="A painless, self-hosted, all-in-one software development service."
```

Comment:
An all-in-one software development platform
(Platform should be better IMO, and please remove the trailing period.)